README.rst

```diff
@@ -71,9 +71,9 @@ You can deploy plugins using CDAP CLI::
 
 Example for loading Cassandra Plugin (from cassandra-plugins directory)::
 
-  > load artifact target/cassandra-plugins-1.0-SNAPSHOT-batch.jar \
+  > load artifact target/cassandra-plugins-1.0-0-batch.jar \
          config-file resources/cassandra-plugin-batch.json
```

Comment:
seems like these filenames are wrong. the ones in the directory have the version at the end.